src/templates/errors.html

```diff
@@ -4,8 +4,8 @@
   <p class="help-block" ng-show="formioForm[componentId].$error.number">{{ component.label || component.placeholder || component.key }} {{'must be a number' | formioTranslate:null:builder}}.</p>
   <p class="help-block" ng-show="formioForm[componentId].$error.maxlength">{{ component.label || component.placeholder || component.key }} {{'must be shorter than' | formioTranslate:null:builder}} {{ component.validate.maxLength + 1 }} {{'characters' | formioTranslate:null:builder}}.</p>
   <p class="help-block" ng-show="formioForm[componentId].$error.minlength">{{ component.label || component.placeholder || component.key }} {{'must be longer than' | formioTranslate:null:builder}} {{ component.validate.minLength - 1 }} {{'characters' | formioTranslate:null:builder}}.</p>
-  <p class="help-block" ng-show="formioForm[componentId].$error.min">{{ component.label || component.placeholder || component.key }} {{'must be higher than' | formioTranslate:null:builder}} {{ component.validate.min - 1 }}.</p>
```

Comment:
I think this is right. The phrase ```must be higher than``` is good. So, if the min is 3 and max is 5, that means that the possible range is 3 to 5. If the value is 2, then this would say ```must be higher than 2``` which is correct. This change would make it say ```must be higher than 3```, when really it should say ```must be greater than or equal to 3```. The reason we picked the previous one is it is shorter to say.